gulpfile.js

```diff
@@ -0,0 +1,21 @@
+var gulp = require('gulp');
+var less = require('gulp-less');
+var path = require('path');
+var minifyCSS = require('gulp-minify-css');
+var rename = require('gulp-rename');
+var sourcemaps = require('gulp-sourcemaps');
+ 
+gulp.task('css', function () {
+  return gulp.src('./jupyter_notebook/static/style/*.less')
+    .pipe(sourcemaps.init())
+    .pipe(less({
+      paths: [ path.join(__dirname, 'less', 'includes') ]
+    }))
+    .pipe(minifyCSS())
+    .pipe(rename({
+            suffix: '.min'
+        }))
+    //.pipe(sourcemaps.write())
```

Comment:
If your intention is to not write the source maps, you can remove `sourcemaps.init()` above and the require.  You could also use a command line parsing library like yargs to make sourcemaps conditional.  i.e. `gulp css --dev` http://stackoverflow.com/a/23038290